src/lib/utils/resource-loader.ts

```diff
@@ -109,6 +113,46 @@ export const tryToLoadFrom = (resourcePath: string): any => {
     return builder;
 };
 
+/**
+ * Check if it is a package with multiple resources.
+ */
+const hasMultipleResources = (resource, type: string) => {
+    switch (type) {
+        case TYPE.connector:
```

Comment:
@sarvaje Can you add more comments here explaining why these things are done. Thanks!